src/meta/processors/job/AdminJobProcessor.cpp

```diff
@@ -142,5 +111,49 @@ void AdminJobProcessor::process(const cpp2::AdminJobReq& req) {
   onFinished();
 }
 
+nebula::cpp2::ErrorCode AdminJobProcessor::addJobProcess(const cpp2::AdminJobReq& req,
+                                                         cpp2::AdminJobResult& result) {
+  auto cmd = req.get_cmd();
+  auto paras = req.get_paras();
+
+  // All jobs here are the space level, so the last parameter is spaceName.
+  if (paras.empty()) {
+    LOG(INFO) << "Parameter should be not empty";
```

Comment:
Prefer log with specific operation, this is too generic. 
If following logs metion this operation, Ignore this is okay.

Reply:
Good point.
The following pr will move the last `spaceName` of parameter out.
Different `AdminJobOp` operations, check the validity of their own parameters.
Therefore, the following pr will uniformly change this error message to make the error message more clear.